cypress/integration/types/constants.ts

```diff
@@ -81,7 +81,7 @@ export const actionsButton = "button[aria-label=Actions]";
 export const duplicateErrMsg = "ERROR: duplicate key value violates unique constraint";
 export const duplicateEmail = "Please use a different email address.";
 export const duplicateName =
-    "An application with this name already exists. Please use a different name.";
+    "A business service with this name already exists. Please use a different name.";
```

Comment:
This might result in application duplicate name test failure .
You can rename this locator to "duplicateApplication" and

create another locator named:
"duplicateBusinessService"